cheribsd/qemu/Dockerfile

```diff
@@ -1,8 +1,16 @@
+# Staged build: import, decompress, and convert disk images (and kernel):
 FROM ctsrd/qemu-cheri:latest
+COPY ./cheribsd-*.img.xz ./kernel.xz /cheri-sdk
+RUN unxz /cheri-sdk/kernel.xz
+RUN for i in /cheri-sdk/cheribsd-*.img.xz; do \
+  unxz $i; i=${i%.xz}; \
+  /cheri-sdk/bin/qemu-img convert -o compression_type=zlib -O qcow2 -c $i ${i%.img}.qcow2; \
```

Comment:
zstd?

Reply:
D'oh, that survived my local testing with older images.  Thanks.